testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveCoercion.java

```diff
@@ -881,11 +881,11 @@ private static List<List<?>> extract(List<TrinoArray> arrays)
 
     public static class ColumnContext
     {
-        private final String hiveVersion;
+        private final Optional<String> hiveVersion;
```

Comment:
> Make version optional in ColumnContext


can this commit include some description on why?
(sorry, i failed to infer this myself)